crates/sui/src/sui_commands.rs

```diff
@@ -88,6 +88,8 @@ pub enum SuiCommand {
             help = "Creates an extra faucet configuration for sui-test-validator persisted runs."
         )]
         with_faucet: bool,
+        #[clap(long, help = "create new validator key pairs on network genesis")]
+        override_validator_keypairs: bool,
```

Comment:
optional: s/override_validator_keypairs/generate_validator_keypairs

Reply:
Going to leave as is, just because this feature doesn't work to just generate validator keypairs if they are left unset in the genesis.yaml, as the keypairs are not optional in the `GenesisConfig` struct. In a future refactor I can hopefully make those fields optional